### DISCUSSION TITLE
请问uniapp打包怎么改成这个rsbuild，后台element ui的项目怎么改成这个打包，谢谢

### DISCUSSION THREAD

**Discussion by brisk21** [web-infra-dev/rsbuild#4085] [Q&A]
公司的项目是基于uniapp开发的，比较庞大，打包很慢，可以改成这个rsbuild吗，然后后台管理是element ui的，可以改成这个打包吗，都太慢了

**Comment by chenjiahan** [comment]
Rsbuild does not currently support uniapp. Element ui can be used with Rsbuild.
